integr8ly/launcher-booster-catalog
# Add Aggregation booster to Red Hat category and lock reference for 1.2.2

## Motivation
JIRA: https://issues.jboss.org/browse/INTLY-1268

**Important:** this needs to be cherry-picked to `integreatly-1.1.0` after merging.

## Verification Steps
Change `launcher` config map in launcher namespace.
Set `launcher.backend.catalog.git.ref` key to `INTLY-1268`
If you are running older version, also set `launcher.backend.catalog.filter` to `booster.version.id.indexOf('redhat')>=0`
Change `launcher-frontend` image tag to `7b1c127` and wait for deployment to finish.
Open Launcher, login and start new project. 
You should be able to select *Aggregation* mission and *Fuse* runtime in the *Mission & Runtime* step.


## Checklist:
